civis/tests/test_futures.py

```diff
@@ -155,9 +164,32 @@ def test_set_api_result_failed(self, mock_subscribe, mock_api):
         api_result = mock.Mock()
         api_result.state = 'failed'
 
-        result = SubscribableResult(poller, (1, 2))
+        result = CivisFuture(poller, (1, 2))
         result._set_api_result(api_result)
         assert mock_pubnub.unsubscribe_all.call_count == 1
         assert result._state == 'FINISHED'
         with pytest.raises(CivisJobFailure):
             result.result()
+
+    @pytest.mark.skipif(not has_pubnub, reason="pubnub not installed")
+    @patch(swagger_import_str, return_value=civis_api_spec_channels)
+    @patch.object(CivisFuture, '_subscribe')
+    def test_overwrite_polling_interval_with_channels(self, *mocks):
+        future = CivisFuture(lambda x: x, (1, 20), polling_interval=5)
+        assert future.polling_interval == _LONG_POLLING_INTERVAL
+
+    @pytest.mark.skipif(not has_pubnub, reason="pubnub not installed")
+    @patch(swagger_import_str, return_value=civis_api_spec_base)
+    @patch.object(CivisFuture, '_subscribe')
+    def test_polling_interval(self, *mocks):
+        # This test uses a different swagger spec than the other tests so it
+        # should clear the cached values before and after
```

Comment:
This test is the case when the channels endpoint isn't available and we fall back on polling, correct? Could you state that in this comment? Also, please add an `assert not hasattr(future, '_pubnub')`.